plugins/ConditionalRestrictions.py

```diff
@@ -161,6 +200,33 @@ def way(self, data, tags, nds):
     if err != []:
       return err
 
+  def isLikelyOpeningHourSyntax(self, condition):
+    # Use a scoring system to determine the likelyness of the condition being time/date based
+    # Not perfect, i.e. 'Mar' and '24/7' will fall through (and bad cases like JAN-APR are thus also not detected)
```

Comment:
You can just add a case for `24/7`.

Reply:
Done! (Although it'll always validate as true since it's a hard-coded, valid value)